mdx-web/src/main/java/com/mx/web/mdx/controllers/AccountsController.java

```diff
@@ -91,6 +91,12 @@ public final ResponseEntity<MdxList<Transaction>> listTransactions(@PathVariable
     return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
   }
 
+  @RequestMapping(value = "/users/{userId}/accounts/{id}/transactions/pending", method = RequestMethod.GET)
+  public final ResponseEntity<MdxList<Transaction>> pendingTransactions(@PathVariable("id") String accountId) throws Exception {
+    AccessorResponse<MdxList<Transaction>> response = gateway().accounts().transactions().pending(accountId);
+    return new ResponseEntity<>(response.getResult().wrapped(), createMultiMapForResponse(response.getHeaders()), HttpStatus.OK);
+  }
```

Comment:
Please add a unit test to `AccountsControllerTest`.

Reply:
Addedd

Reply:
Did the test get removed? I don't see it anymore in the diff.